README.md

```diff
@@ -1,4 +1,145 @@
-# Verify Local Matching Service Example
+# Example local matching service for GOV.UK Verify
 
-This is an example of local matching service that could be used by relaying parties as a reference when building their matching service.
+This is an example local matching service built by the GOV.UK Verify team to demonstrate how a local matching service works and help services building their own. Every government service connecting to Verify must build their own local matching service.
```

Comment:
I feel like `must build their own` sounds a bit anti-collaborative. Maybe just `must build a local matching service` as in an ideal world we would actually have us and RPs sharing code/helping each other 😄